org.mixedrealitytoolkit.input/Tracking/TrackedPoseDriverExtensions.cs

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Mixed Reality Toolkit Contributors
+// Licensed under the BSD 3-Clause
+
+using UnityEngine.InputSystem.XR;
+using UnityEngine.XR;
+
+namespace MixedReality.Toolkit.Input
+{
+    /// <summary>
+    /// Extensions for Unity's <see cref="TrackedPoseDriver"/>
+    /// </summary>
+    public static class TrackedPoseDriverExtensions
+    {
+        /// <summary>
+        /// Gets the tracking state of the <see cref="TrackedPoseDriver"/>. If the tracking state is not available, returns false.
+        /// </summary>
+        public static bool TryGetTrackingState(this TrackedPoseDriver driver, out InputTrackingState state)
+        {
+            state = InputTrackingState.None;
+            var trackingStateAction = driver.trackingStateInput.action;
+            if (trackingStateAction == null || trackingStateAction.bindings.Count == 0)
+            {
+                return false;
+            }
+
+            if (trackingStateAction.enabled)
```

Comment:
Should this be `if (!trackingStateAction.enabled)`?

Reply:
I think this is correct as it is because TrackedPoseDriver::ReadTrackingState() `"// Treat a disable action as the default None value for the ReadValue call"` as in:
![image](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/assets/84108471/70f4f693-a69e-4ba9-9c7f-5bf29cc4d78b)
Meaning that a disabled `trackingStateAction` can be queried for tracking state and the result be InputTrackingState.None and since this is a `TryGetTrackingState` method then this logic will return true (can be queried) and the value will be InputTrackingState.None.  @AMollis Do you concur?

Reply:
Ooops!!!  typo will fix